src/app/components/plugin-manager-proxy.js

```diff
@@ -16,12 +16,12 @@ class PluginManagerProxy {
       registry.get('compilersartefacts').api['__last'] = new CompilerAbstract(languageVersion, data, source)
       event.trigger('sendCompilationResult', [file, source, languageVersion, data])
     }
-    instance.event.on('compilationFinished', this._listener)
+    instance.events.on('compilationFinished', this._listener)
```

Comment:
if that instance is the solidity compiler, i think it implement `event`

Reply:
When testing, this instance was the `CompileTab`. I switched to `events` in `CompileTab`. But can it be also the solidity compiler from remix lib ?

Reply:
> but can it be also the solidity compiler from remix lib ?

 not it is not the same